tests/ui/step/SubStepTest.cpp

```diff
@@ -75,4 +75,22 @@ TEST_F(SubStepTest, shouldReadDate) {
             .WillOnce(Return(task_));
     EXPECT_TRUE(std::dynamic_pointer_cast<SubStepLabel>(
             step.execute(*context_)));
+}
+
+TEST_F(SubStepTest, shouldReadLabels) {
+    auto step = SubStepLabel(factory_, view_);
+
+    EXPECT_CALL(*view_, ReadLabels(_))
```

Comment:
need to test, that command_name() result was moved to ReadLabels().

Reply:
Fixed